src/components/ui/Box.tsx

```diff
@@ -0,0 +1,16 @@
+import { styled } from "@fast-styles/react";
+import { globalStyles } from "@src/styles/globalStyles";
+import { View } from "react-native";
+
+const Box = styled(View, {
+  display: "flex",
+  width: "100%",
+  padding: 16,
+  backgroundColor: globalStyles.colors.white,
+  borderRadius: 16,
+  borderStyle: "solid",
+  borderWidth: 1,
+  borderColor: globalStyles.colors.neutral[100],
```

Comment:
can we pass the colors as props so they can be customized?

https://fedemartinm.github.io/fast-styles/docs/basics/style-props

Reply:
Ok sure, added here with default values: https://github.com/LangaraBeetles/weaup-react-native-app/pull/42/commits/896746e2c12e512d4351dadd13d4dcb5651f870d